test/integration/WireMock.spec.ts

```diff
@@ -32,15 +32,14 @@ describe('Integration with WireMock', () => {
         res.end();
     };
 
-    beforeAll((done) => {
+    beforeAll(() => {
         // Create webhook server. This will be used by wiremock.
         const app = express();
         app.use(express.json());
         app.get('/webhook', webhookGetHandler);
         app.post('/webhook', webhookPostHandler);
-        server = app.listen(WEBHOOK_PORT, done).on('error', (e) => {
+        server = app.listen(WEBHOOK_PORT).on('error', (e) => {
```

Comment:
Doesn't this cause a leak if there is a startup error?

Reply:
calling `done` was unreachable code and TS now complains about it with the updates

`fail` will short circuit, so if you do want to execute `done`, it will have to be somewhere else

Reply:
Ahh I see, so fail should quit the test. Makes sense.